JSDemos/Demos/Charts/CustomAnnotations/description.md

```diff
@@ -1 +1,17 @@
-To customize an annotation, declare the SVG markup in the annotation [template](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#template). Inside the template, you can access the annotation being customized. 
\ No newline at end of file
+Annotations are containers for images, text blocks, and custom content that display additional information about the visualized data. To create custom annotations, specify the following Chart properties.
+
+## annotations[]
+
+The [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) array specifies a collection of annotations. This demo uses the data source array to create this collection. Each object has an [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) to make the component display annotations relative to specific arguments and a [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object that stores data from data source.
```

Comment:
> This demo uses **the** data source array to create this collection.

Which data source exactly? I'm questioning your choice of article here.

> Each object has an [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) to make the component display annotations relative to specific arguments

I cannot say I understand this. Did you mean that the mentioned argument allows you to position annotations in different ways? If yes, I believe we need to clarify this further.

> a [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object that stores data from **data source**.

An article is missing.

Reply:
I feel like I need to explain how the **argument** property works.
So there are multiple different ways to specify where the annotations should be displayed. You can set coordinates for each annotation, you can display annotations on axes, and you can anchor them to a chart element. Charts have argument axis and value axis. If you want to anchor annotation to a specific chart point, then you need to specify both **value** and **argument** properties (chart coordinates). And if you need to anchor annotation to a series point (series is group of related points, like in this demo there are multiple bars, the whole group of these bars is a series, and each bar is a series point), you need to specify both **argument** and **series** properties, since each series point represents one argument. In this demo, the **series** property is already specified in the **commonAnnotationSettings** object, so it's the same series for all annotations. But each annotation has different argument value (a different series point). So that's why it's necessary to specify the **argument** in the **annotations** array, so the component knows where to display the annotations. If you don't specify the **argument**, the Chart shows annotations on the series, but near the value axis.

I tried not to explain all of it in the demo, since all of this information can be found in the documentation, so I tried to squeeze all of this into one sentence, but I guess I should expand it instead.

Reply:
Alright, now I understand your point. However, the sentence in question looks as if something is amiss. I'll offer you my version and you'll see if it suits you.